storage/src/mmr/benches/bench.rs

```diff
@@ -1,11 +1,11 @@
-use commonware_storage::mmr::{verify_proof, verify_range_proof, Hash, InMemoryMMR, Sha256Hasher};
+use commonware_cryptography::{Digest, Hasher, Sha256};
+use commonware_storage::mmr::{verify_proof, verify_range_proof, InMemoryMmr};
 use criterion::{criterion_group, criterion_main, Criterion};
 
 fn bench_build_mmr(c: &mut Criterion) {
-    let element = Hash::new(&[100u8; 32]);
+    let element = Digest::from_static(&[100u8; 32]);
     c.bench_function("build_mmr", |b| {
```

Comment:
To play nicely with the benchmarks site, we need to use the format of 1 benchmark per file (where the name of the file is the name of the benchmark).

You can see how this looks here: https://github.com/commonwarexyz/monorepo/tree/main/cryptography/src/bls12381/benches

The `bench.rs` file just lists all the sub-benchmarks: https://github.com/commonwarexyz/monorepo/blob/main/cryptography/src/bls12381/benches/bench.rs

Reply:
done